File: app/src/main/java/com/univreview/fragment/login/UserAuthCompletedFragment.java
package com.univreview.fragment.login;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.univreview.R;
import com.univreview.fragment.BaseFragment;
import com.univreview.util.Util;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * Created by DavidHa on 2017. 3. 22..
 */
public class UserAuthCompletedFragment extends BaseFragment {
    @BindView(R.id.complete_btn) Button completeBtn;

    public static UserAuthCompletedFragment newInstance(){
        UserAuthCompletedFragment fragment = new UserAuthCompletedFragment();
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        super.onCreateView(inflater, container, savedInstanceState);
        View view = inflater.inflate(R.layout.fragment_user_auth_completed, container, false);
        ButterKnife.bind(this, view);
        toolbar.setVisibility(View.GONE);
        rootLayout.addView(view);
        completeBtn.setOnClickListener(v -> getActivity().finish());
        return rootLayout;
    }
}

File: app/src/main/java/com/univreview/fragment/login/UserAuthCompletedFragment.kt
package com.univreview.fragment.login

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import butterknife.ButterKnife
import com.univreview.R
import com.univreview.fragment.BaseFragment
import kotlinx.android.synthetic.main.fragment_user_auth_completed.*

/**
 * Created by DavidHa on 2017. 11. 2..
 */
class UserAuthCompletedFragment : BaseFragment() {
    companion object {
        fun getInstance(): UserAuthCompletedFragment {
            return UserAuthCompletedFragment()
        }
    }


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        super.onCreateView(inflater, container, savedInstanceState)
        val view = inflater!!.inflate(R.layout.fragment_user_auth_completed, container, false)
        ButterKnife.bind(this, view)
        toolbar.visibility = View.GONE
        rootLayout.addView(view)
        completeBtn.setOnClickListener { activity.finish() }
        return rootLayout
    }
}